Proposal for advancing issues while keeping editors in loop (#182)
solid/process: README.md

Candidate proposals to the [Solid Specification](https://github.com/solid/specification) submitted for review go through an editorial process before they are accepted.
Candidate Proposals to change the Solid Specification must be submitted for editorial review before they are accepted, along with the results of any votes taken.

> To help broad consensus form, it is suggested that proposals are brought to the Editor's attention at some defined contact points during its life cycles. These contact points are: 

Reviewer: ```suggestion
To help broad consensus form, it is suggested that each proposal be brought to the Editor's attention at some defined contact points during its life cycle. These contact points are: 
```

> To help broad consensus form, it is suggested that each proposal be brought to the Editors attention at some defined contact points during its life cycle. These contact points are:

Reviewer: ```suggestion
To help broad consensus form, it is suggested that each proposal be brought to the attention of the Editors at some defined contact points during its life cycle. These contact points are:
```
or `Editors' attention`

> To help broad consensus form, it is suggested that each proposal be brought to the Editor's attention at some defined contact points during its life cycle. These contact points are: 

Reviewer 2: Minor suggestion in changing the wording to - "brought to the attention of the Editors", so that it is clearly plural and meant to imply they notify the whole group rather than one singular person. Another thought would be to explain what might be the best way to reach them (i.e. some combination of mailing list, gitter channel, github mentions).

Author: OK, good, please have another look

> 1. When a problem with the specification is first encountered, an [issue is opened](https://github.com/solid/specification/issues/new). 

Reviewer 2: I wouldn't classify all reasons to change the specification as "problems", so perhaps something along the lines of "When a reason to change or extend the specification"?

Author: Yeah, good one.

> 1. When at first a problem with the specification is encountered, an [issue is opened](https://github.com/solid/specification/issues/new). 

Reviewer: ```suggestion
1. When a problem with the specification is first encountered, an [issue is opened](https://github.com/solid/specification/issues/new). 
```

> 1. Panels may create or adopt issues, and panels should notify Editors that discussion has started.

Reviewer: ```suggestion
1. Panels may create or adopt issues, and Panels should notify Editors that such discussion has started.
```

> 1. When a Panel reaches rough consensus, Panels should notify Editors and indicate in an informal text what the consensus entails and who participated in forming it.

Reviewer: ```suggestion
1. When a Panel reaches rough consensus on an issue, the Panel should notify Editors, indicating in informal text what the consensus entails and who participated in forming it.
```

> 1. When a Panel starts drafting the text of a proposal Panels should notify Editors.

Reviewer: ```suggestion
1. When a Panel starts drafting the text of a proposal, the Panel should notify the Editor(s).
```

> The above is not required and superfluous for small changes, but for larger issues, it is likely to speed up the overall process.

Reviewer: ```suggestion
The above is not required and may be superfluous for small changes, but it is likely to speed up the overall process for larger issues.
```
